notebooks/AsymmetricErrors.ipynb

```diff
@@ -0,0 +1,577 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "ui.plot_scatter(result['pl.gamma'], result['pl.ampl'], xlabel=r'$\\Gamma$', ylabel='ampl')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "70f782e9-c851-4432-bf3d-3dcc2f6c1a89",
+   "metadata": {},
+   "source": [
+    "We can compare the fit result from earlier to that from the resampled data. In this case they do not agree, which is interesting but I do not know enough about this data set to understand what this means!"
+   ]
```

Comment:
Instead of saying you don't know, say that the conf is using symmetric errors as in the fit, so no information about the asymmetric errors for the data points. `conf` and 'reg_proj' will come with a  too narrow range, while the `resample_data` function uses the information about the data.

Reply:
I've reworked this paragraph a little bit, and added a little bit about using the sampled values to estimate useful information.